bindings/ruby/extconf.rb

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+require 'mkmf'
+
+$LDFLAGS << ' -lc++'
```

Comment:
I just compiled with `clang++` and there were 6 warnings and 11 errors. Also this message:

    clang: warning: treating 'c' input as 'c++' when in C++ mode, this behavior is deprecated